core/services/offchainreporting/OffchainReportingAggregator.go

```diff
@@ -0,0 +1,233 @@
+package offchainreporting
+
```

Comment:
Nit: rename `OffchainReportingAggregator` to `Contract` or `OCRContract` and lower case the file name?